common/app/model/CrosswordGridModel.scala

```diff
@@ -9,15 +9,13 @@ import com.gu.contentapi.client.model.v1.{
 import model.{CrosswordPosition, Entry}
 
 import Function.const
-import scala.collection.compat.immutable.LazyList
 
 trait CrosswordGridDataOrdering {
   implicit val positionOrdering = Ordering.by[CrosswordPosition, (Int, Int)](position => (position.y, position.x))
 }
 
 trait CrosswordGridColumnNotation {
-  val columnsByLetters =
-    (('A' to 'Z').toList.zip(LazyList from 0) map { case (letter, number) => (number, letter) }).toMap
+  val columnsByLetters = ('A' to 'Z').toList.zipWithIndex.map(_.swap).toMap
```

Comment:
We used `LazyList` as part of the [Scala 2.13 upgrade because `Stream` got deprecated](https://github.com/guardian/frontend/pull/25425/commits/027bd3211a61f2f474ad2bf960bed846af1203ad). However, we can re-write the code using `zipWithIndex`

![Screenshot 2022-10-04 at 13 08 55](https://user-images.githubusercontent.com/19683595/193839357-5f4da2f9-ce56-4d1b-9e0f-3caab00cba3e.png)

Reply:
I would be concerned about using this if it were the wrong thing, but this code is way nicer!